pkg/querier/frontend/frontend.go

```diff
@@ -192,11 +192,7 @@ func (f *Frontend) handle(w http.ResponseWriter, r *http.Request) {
 		hs[h] = vs
 	}
 	w.WriteHeader(resp.StatusCode)
-
-	if _, err = io.Copy(w, resp.Body); err != nil {
-		server.WriteError(w, err)
-		return
-	}
+	io.Copy(w, resp.Body)
```

Comment:
We cannot do much about errors returned here, but doing nothing seems like too little. We could log it, which may have benefit of being logged with traceID so it can be tracked back to specific clients (in case someone is complaining).

Reply:
Ah, sorry I see it's logged by middleware. Ignore.